src/components/util/saveConfirmedInputValueToLottoNumberData.js

```diff
@@ -0,0 +1,34 @@
+const { LottoNumberData } = require('../lotto-data/LottoNumberData');
+
+const MONEY = 1000;
+const OBJECT = 'object';
+const BONUS = 45;
+
+function saveConfirmedInputValueToLottoNumberData(value) {
```

Comment:
클래스명을 통해서 어떤 기능을 하는지 알기는 쉬우나, 클래스 네이밍이 너무 길어서 가독성이 좀 떨어지는 것 같아요!
조금 더 함축적으로 나타 낼 수 있지 않을까 싶습니다!! 
하단과 같은 네이밍을 조심스럽게 추천드려봅니다..! (나타내는 의미가 좀 다를 수 있을 수도 있습니다)
```suggestion
function returnConfirmedData(value) {
```
효과적인 네이밍을 위해 링크 하나 첨부해드립니다!!
https://remotty.github.io/blog/2014/03/01/hyogwajeogin-ireumjisgi/